packages/provider-elasticsearch/src/example-types/results/highlighting/request.js

```diff
@@ -190,7 +190,7 @@ export let getRequestHighlightFields = (schema, node) => {
   // TODO: `highlightOtherMatches` is an undocumented configuration value that we
   // are currently using to work around performance issues when highlighting
   // fields not included in the node.
-  if (!node.highlight?.highlightOtherMatches) {
+  if (!node.highlight?.highlightOtherMatches && _.isArray(node.include)) {
```

Comment:
Small thing, but we should probably prefer the `Array.isArray` built-in here.

Reply:
I agree in general but I think we should keep the pattern consistent and switch to it everywhere once.

Reply:
There shouldn't be any harm here, but okay.